Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -1000,3 +1014,38 @@ void wpe_view_set_opaque_rectangles(WPEView* view, WPERectangle* rects, guint re
     if (viewClass->set_opaque_rectangles)
         viewClass->set_opaque_rectangles(view, rects, rectsCount);
 }
+
+/**
+ * wpe_view_set_gesture_controller:
+ * @view: a #WPEView
+ * @controller: a #WPEGestureController
+ *
+ * Set @controller as #WPEGestureController of @view
+ */
+void wpe_view_set_gesture_controller(WPEView* view, WPEGestureController* controller)
+{
+    g_return_if_fail(WPE_IS_VIEW(view));
+
+    auto* viewClass = WPE_VIEW_GET_CLASS(view);
+    if (viewClass->set_gesture_controller)
+        viewClass->set_gesture_controller(view, controller);
```

Comment:
And this should be a normal setter, but we should take a reference, not adopt it.